clean-lumber.lic

```diff
@@ -24,7 +24,7 @@ class CleanLumber
   end
 
   def process_size(args, size)
-    while bput("get #{args.type} #{size} from my #{args.source}", 'You get', 'You carefully remove', 'What were you') != 'What were you'
```

Comment:
During testing, I was having issues with the comparisons not working, and switching the order got them to work -- I'm not sure why  If it's a big deal, I can go back and try them the original way again to try to get them working like that.